standard_forms/supplier-return/latest/report-manifest.json

```
@@ -0,0 +1,14 @@
+{
+  "is_custom": false,
+  "version": "2.6.5",
+  "code": "supplier-return",
+  "context": "OUTBOUND_SHIPMENT",
+  "sub_context": "SupplierReturn",
```

Would change version to match app version and context should be supplier return (: else it shows up in the Outbound selection~

```suggestion
  "version": "2.13.0",
  "code": "supplier-return",
  "context": "SUPPLIER_RETURN",
```